src/reducers/scenes/CreateWalletReducer.ts

```diff
@@ -17,25 +17,10 @@ export interface AccountActivationPaymentInfo {
 }
 
 export interface CreateWalletState {
-  handleActivationInfo: HandleActivationInfo
```

Comment:
The type definition `HandleActivationInfo` should probably move out of this file and into `CreateWalletActions.ts` or whichever file uses it now.